### DISCUSSION TITLE
SageTV - deleted a TV lineup by mistake and now can't add it

### DISCUSSION THREAD

**Discussion by Swagger200** [OpenSageTV/Community#155] [General]
Using SageTV 9 servers and clients. While setting up a new server I accidentally deleted a TV lineup supplied by Schedules Direct from my Hauppauge HD-PVR video source. I tried to add it back by going through the Country / Postal Code search function in SageTV source setup, but when I enter my valid postal code, SageTV says it is invalid and will not proceed. I have tried all combinations of space between, no space, caps and not caps. Is anyone aware of a workaround, perhaps by using a Sage properties file from another working server? Thanks

**Comment by emveepee** [comment]
Does your postal code work on zap2it.com.  If so what is your postal code so I can check the API?

**Reply by Swagger200** [reply]
Yes it does. L4G2W6

**Reply by emveepee** [reply]
It is definitely valid on SchedulesDirect  Are you using a VPN or running multiple servers?  SD needs tokens to be consistent and no one modified SageTV logic to handle that.

**Reply by Swagger200** [reply]
No VPN but I have 5 servers

**Reply by emveepee** [reply]
Stop the others and start this one and see if that helps, assuming they are on the same IP.

**Reply by Swagger200** [reply]
Yes they are. Do you know if the SD token is saved in the SageProperties file? I ask because I have been able to work around this by copying over the SageProperties file from another working server. I am worried now that if I shut down the other servers, then a new install will create a new token and the other servers will stop working,

**Reply by emveepee** [reply]
I don't know SageTV but SD tokens are temporary and have a lifetime so they to need to be generated dynamically and there is not much value in persisting them to disk.     When one server initiates a token it will invalidate all the other servers' tokens.

Running five simultaneous servers seem insane to me unless you are developing.
